sys/include/net/nanocoap.h

```diff
@@ -1,16 +1,102 @@
 /*
  * Copyright (C) 2016-17 Kaspar Schleiser <kaspar@schleiser.de>
  *               2018 Freie Universität Berlin
+ *               2018 Ken Bannister <kb2ma@runbox.com>
  *
  * This file is subject to the terms and conditions of the GNU Lesser
  * General Public License v2.1. See the file LICENSE in the top level
  * directory for more details.
  */
 
 /**
- * @defgroup    net_nanocoap nanocoap small CoAP library
+ * @defgroup    net_nanocoap Nanocoap small CoAP library
```

Comment:
I made the 'N' upper case here only to support the case sensitive sort of the Doxygen tree view. Otherwise, nanocoap appears at the bottom of the list of Networking sub-modules. I'm not proposing to change case systematically. It just seemed worth it here to make the information easier to find. I don't see a way to change how Doxygen sorts the tree view.

Also, I'd prefer to leave the description as is for now. I might pick a different word than "minimal". :-)

Reply:
I made the 'N' upper case here only to support the case sensitive sort of the Doxygen tree view. Otherwise, nanocoap appears at the bottom of the list of Networking sub-modules. I'm not proposing to change case systematically. It just seemed worth it here to make the information easier to find. I don't see a way to change how Doxygen sorts the tree view.

Also, I'd prefer to leave the description as is for now. I might pick a different word than "minimal". :-)